docs/api/guarding-policy-api.mdx

```diff
@@ -0,0 +1,97 @@
+---
+sidebar_position: 7
+title: Policy Guard API
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+The Policy Guard API allows you to manage policies at the organization level. Policy Guards help enforce centralized control over permissions and roles across specific projects included in the policy.
+For example, you can create  that enforces a specific set of permissions, as role admin can create documents, and role viewer can only read documents, across all projects included in the policy.
+
+Let's see how you can use the Policy Guard API for the example above.
+
+
+## Example API Usage
+
+
+### Create a New Policy Guard Scope
```

Comment:
Add some text to explain the action - 
"First, we'll create a new policy guard scope in order to....